src/book.rs

```diff
@@ -42,12 +42,7 @@ pub struct Book {
     pub market: Address, /* the address of the Tracer market */
     pub bids: BTreeMap<U256, VecDeque<Order>>, /* buy-side */
     pub asks: BTreeMap<U256, VecDeque<Order>>, /* sell-side */
-    #[serde(
-        serialize_with = "from_hex_se",
-        deserialize_with = "from_hex_de",
-        rename = "LTP"
-    )]
-    pub ltp: U256, /* last traded price */
+    pub ltp: U256,       /* last traded price */
```

Comment:
@ScaredIbis just commented to double check. Is this actually a hex string and not a uint string? PR description says hex string but this seems to indicate U256 string?

Reply:
yeah it comes out as a hex string, I think the ethereum types U256 type defaults to serialising to a hex string